packages/services/hmi-server/src/main/resources/META-INF/resources/index.html

```diff
@@ -1,285 +1,359 @@
40sbdfw5qvmS8ZfoVHxvuD/O63u/rvhLUxPfe4Kblzr8fOky+t7zfHnJCt/ge4u81HHY8qW7Qb4U8L2ZL80K5iXo4/L4EuSlb3LfG/lSh58vUd6BvLQoo4+z/hLyJfS9Hzzw5CXfvrfYq9R7Avd0XlpkfG/gS8ZfomP6OHuyfW/dx423fOnYCXp/Ab4E+96Ql1ZuBb4Ee5V5+97oey90fe/8++OWx9wfh/eh3Mi6Pw59b5WXsu+PM3lpWK7vbfe9Rwf40oeo++OWevkS+97gL1W7JxC+Pw7zEvjeZfa9gS8dSd8fNzZq37slyJd64v1x05zn43BPQPClscCXou+PGwj3xx2E+1Dg/jjeE8i+P07mJfoz7uPi74/jfe/g/XHaX7J8Sex717j3x5X07ADve6PvTQnpZ/j+uDrBl8D3ru7+uFrf/XHAl+xLqKf7w9u5xeg1hWH4nxlK//SfYgYd6jSKthhqVOs8dIw6tWIcRtFBgiiREpmWC+IYx0QcSp3iEGdtggtx0QSJYOJwIRFx5cKdO1cufXut9a93r/Wub6+9x2EnGEVnBknfPN+z37c1mt+PG5a/RP1LyEtz2Y+D7/1rfb40fLDzvfeRB3zJ5KUrqvjS9eBLeD/OHBJV39vkP+d7r+vypUReGhziPgE9Lw2ZvKT0Lym+9y3lvHS5y0sbpbQbfAl5Ke17H5/lS+j3vjTmS7d6vvQR7nHUvxTc4zZ7vnS+50v3NOVLs+gTkB/S9+PAl7YrfOmj2Y+i/bjv4HvLH+Evzab50qzme3+k5qVvbF7a7u5x8uh8CXkp5EufmLy0I/SXLF86H3np5tBfupf6veVxvveLpk/gLN6Pc753293jBsp7KMUNzu7trkz5S1u2uD4BKzC5vBTypansftyLuMcp+3HsL+X3UNR+b71PAL73mOkTWK7c45L93u2QL61AXsrvx6FPYCy3H9foHtfme1wH+3HEl8I+gXA/bu//zffu8f3eR+j7ceBLlb73FnePWxb53suWh32V/fC9E/txK81+HPneE4HvTXwp8L25r3L/4B7n9uO4T0ACU+V+3LwW+FJiP04edT9OyUu2T+C4xH6c/K6Ul9L7cUG/92VBv/exUf9SX7dPYF+NL+1bcz/uRO731n1v9AkQXzIiftr33q1398MWtLT+pYP5Hpffj7smux8nT91+7/f93u6VJjANW750OPIS+NL1tB9HeYn5EvLS8Oq7Xwr6BNb5/qUiTGA/zt7jjgry0pCSl0a433uo5HsvTecl+YH8Pc75S1u797iIL3FeYr7UzUuSvNAnwHwJfQIv6L63/BG+N/Ml/R43U85LM3Fe2pzjS4+S7839S+4eN+v40nfo98Y9DnzJ9gnA967kSxKHZjTf+1343qW8ZPnSWarvzXwJeenaKt9b+r01voR+7xeN7236Ku+NfG/5EHkp2I/L+t7SJ2DvcfL4/qWwT+Brd4/70fUJRP7SxQFfOm+q4X7cG3PzvVfkfW+9TyB/j+P9OPQJNPO9D0rsx1X63vOJLym+twlMq8ZzfQLy36PmftwrOl/S9+POjPKS3Y9b0N2PWxTxJdzjprAfF9/jxkt5Ked7O1O/ej/uKed7q3xpSdS/dGQD3xt8Ce/HKfc4pU+gB3kJ+3He9z4o43sjL9F+XNL37tH240xdJe/H9QR9Arwf1+t8b+QlvB/XaD9uYY39uP1KeSnRJ7BfKi/t5u5xkjtb6n7cYGY/bii/H3e0ypca9XtfafdQuv3eRWAK+iqXBnwJeemnRF5aKnyJ73GnPHP3GUq/9zrmS5KXEr73Nem+yiICRXzJ5qWlWl5S+gQMe3J8SUJQ4C/FfOn4ZF6S6x3zJZOX8H4c8SXKS+R7nwq+VOpf2mnzEvtLzX1vtd9beIq5x12i5iXDlz6UwBTlpe/4/Th5kJfcPY77vWve44q8RL73GvK9qU/g05AvfRzxpVflHqfyJfjedI9zfGna5SW5x8V8aSXxJZsS4HvbOoGVhjWV8hJ8b+0eJ7/ORH0C8Jea93tP1PW9v4r8JQlMFf1LSb60nP2l9Zv6sYdCfMn63gP96n7cyep+3FTJ99b8JclL2I/L9Ql8C99b6RNg3zvbJ9B8P+6VRvtx3l9Cv/e8fTXfe0vX974/4ktPZ/bj0CeAvkrje3O/93rqE1B97yxf0vd2LV8KfG+7h/Klvh93ULAfhz6BgC81248DX6J7nO+rXMh8Cb635CaFL8H35n7vXnuPQ59AeT8u0yewZ3yP6wn4kjyWL5mnxn7cbuj3TvAlcexb5C8pfQJx/9KI34+Tp+F+XB3f+/2SvyTPsM1LVx5u8xL8pYPBl/R+70H5B7p5ye/HgS85fynaj/P+kstL6Pc+BXxJy0sjvB83VOJLS9W8BH/pPt33vsX4S9X3uFsK33vS+d7H4x4X8SV/j4O/9CDyEvhS0l9CXlp7TgO+lO8T+GYW/hL73j4vLa7OSxKYHF5K86XvwJfIX1L4Et/jmC/tMnyJ/KU1ob+0JvKXxPfW/CXkpT8dXzo/9L1vJ99b5UvwvdW8xL43+JJ8nOwTELpk+RLf44Qv6f4S+gR03/t07MdNoK8ys4dSzku0H0e+9zF6n8De3l9ab/oE9H7vDvUJrARfsn0C2f24Ked76/tx7HujrxJ8ifbjcv4S86Wc7019Ao3345bYvPS7jUvymD4B+UUTfQLBftz8nO/9cOIeh/04k5c2oN+75C/12/04ITS0H0e+d+0+AbGXmuzHgS+R753fj2uZPoElni9hP+7Yiv2469An0EG/d0/Qv0S+d0/VflyH+yp7oj6Bnl7aj2O+NPf9OOQl7hPoqH0CyEtpviRfpnyL8j22+B6X2o8b/Ef7cYNz2I9L5iW5x4nvLbxL7V+K+NIfb+f50lGnoE9gyPEledK+t2RFxfcmvpT2l0ZPqn+Pkzyj+t7H2z4BLS/JU+kvfabf4x6ke1w1XzqV8hL4kuovCWCaQV5q7HsX6cDwpaeq7nGSl2YL
```

Comment:
Is it part of the Quarkus license to include this boiler-plate text?  Or can/should it be removed?

Reply:
this is the Quarkus dev page. When you run quarkus in dev mode, you can change configurations on the fly using this interface. It's not in production